validate-microshift/kuttl-test.sh

```diff
@@ -5,12 +5,19 @@ set -x
 
 ROOT="$(readlink -f "$(dirname "${BASH_SOURCE[0]}")/../")"
 
-KUTTL_VERSION="0.10.0"
+KUTTL_VERSION="0.15.0"
 KUTTL="$ROOT/bin/kuttl"
 
+if [ $(uname -p) == "x86_64" ]; then
+    ARCH=x86_64
+else
+    ARCH=arm64
+fi
```

Comment:
A nit: to make this code future proof, we'd better have a case here for supported platforms with an error indication for the unsupported ones.